options.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const tsServerOptionName = 'ts-server';
+const tsProjectOptionName = 'ts-project';
+const sapUserOptionName = 'sap-username';
+const extendedUriFilterOptionName = 'extended-uri-filter';
+const allOptions = [tsServerOptionName, tsProjectOptionName, sapUserOptionName, extendedUriFilterOptionName];
```

Comment:
AFAIS, these `const` parameters are actually an `id`, right? Because we get these elements by using these `names` below by calling `getElementById()`. If so, maybe renaming them like would make much sense:

```suggestion
const tsServerOptionId = 'ts-server';
const tsProjectOptionId = 'ts-project';
const sapUserOptionId = 'sap-username';
const extendedUriFilterOptionId = 'extended-uri-filter';
const allOptionIds = [tsServerOptionId, tsProjectOptionId, sapUserOptionId, extendedUriFilterOptionId];
```

Reply:
renamed with `id` suffix and moved to `constants.js`